### ISSUE TITLE
Summons completely break the mod

### ISSUE THREAD

**Issue by TiamatV** [BarleyFlour/WrathVisualAdjustmentsMod#76]
Anyone else have the issue of summons of any kind breaking the mod? It makes the mod unuseable, and trying to restart the mod requires a complete game restart

[Visual adjustment Crashlog.txt](https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/files/7231937/Visual.adjustment.Crashlog.txt)

**Comment by Dhraiden** [comment]
Yes, I just indicated the same with my own post relaying the same errors generated.

https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/issues/80
